src/main/java/org/opentripplanner/standalone/GraphBuilderParameters.java

```diff
@@ -198,6 +198,18 @@
      */
     public boolean writeCachedElevations;
 
+    /**
+     * When set to true (it is false by default), the elevation module will include the Ellipsoid to Geiod difference in
+     * the calculations of every point along every StreetWithElevationEdge in the graph.
```

Comment:
I think we should note how this relates to the router-config value. I.e., if this value is set to true, should the router-config value be set to false? If they're both set to true will that double the geoid elevation value offset?